src/settings.js

```diff
@@ -474,6 +474,18 @@ var GL_EXTENSIONS_IN_PREFIXED_FORMAT = 1;
 // you will need to manually enable the extensions you need.
 var GL_SUPPORT_AUTOMATIC_ENABLE_EXTENSIONS = 1;
 
+// If true, the function emscripten_webgl_enable_extension() can be called to
+// enable any WebGL extension. If false, to save code size,
+// emscripten_webgl_enable_extension() cannot be called to enable any of extensions
+// 'ANGLE_instanced_arrays', 'OES_vertex_array_object', 'WEBGL_draw_buffers' or
+// 'WEBGL_draw_instanced_base_vertex_base_instance', but the dedicated functions
+// emscripten_webgl_enable_*() found in html5.h are used to enable each of those
+// extensions. This way code size is increased only for the extensions that are
+// actually used.
+// N.B. if setting this to 0, GL_SUPPORT_AUTOMATIC_ENABLE_EXTENSIONS must be set
+// to zero as well.
```

Comment:
if not already done, assert somewhere if these flags conflict? (I think it can be done in emcc.py?)